package.json

```diff
@@ -2,18 +2,18 @@
   "name": "CodiMD",
   "version": "1.5.0",
   "description": "Realtime collaborative markdown notes on all platforms.",
-  "main": "app.js",
+  "main": "lib/app.js",
   "license": "AGPL-3.0",
   "scripts": {
     "test": "npm run-script eslint && npm run-script jsonlint && npm run-script mocha-suite",
-    "eslint": "node_modules/.bin/eslint --max-warnings 0 lib public test app.js",
+    "eslint": "node_modules/.bin/eslint --max-warnings 0 lib public test",
     "jsonlint": "find . -not -path './node_modules/*' -type f -name '*.json' -o -type f -name '*.json.example' | while read json; do echo $json ; jq . $json; done",
     "mocha-suite": "NODE_ENV=test CMD_DB_URL=\"sqlite::memory:\" mocha --exit",
     "standard": "echo 'standard is no longer being used, use `npm run eslint` instead!' && exit 1",
     "dev": "webpack --config webpack.dev.js --progress --colors --watch",
     "heroku-prebuild": "bin/heroku",
     "build": "webpack --config webpack.prod.js --progress --colors --bail",
-    "start": "sequelize db:migrate && node app.js"
+    "start": "tsc && sequelize db:migrate && node built/app.js"
```

Comment:
Sure about `built` as directory? I would rather suggest using `dist`.

Reply:
That is what the [handbook](https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html) used, but the naming is debatable, yeah ;)